listen.py

```diff
@@ -43,6 +43,41 @@ def from_json(cls, j):
             data=j.get('track_metadata')
         )
 
+    @classmethod
+    def from_influx(cls, row):
+        """ Factory to make Listen objects from an influx row
+        """
+        dt = datetime.strptime(row['time'] , "%Y-%m-%dT%H:%M:%SZ")
+        t = int(dt.strftime('%s'))
+        mbids = []
+        if row.get('artist_mbids'):
```

Comment:
@alastair, I wrote it like this to get rid of the internal server errors. What you're suggesting is what was written previously. If some listens do not have artist_mbids, influx will return None in the `artist_mbid` field. Then `row.get('artist_mbids', '')` returns `None` which cannot be split.